spec/scrabble_spec.rb

```diff
@@ -0,0 +1,65 @@
+require './lib/scrabble.rb'
+require 'spec_helper.rb'
+
+describe "Scrabble" do
+  # include Scrabble
+  context "#self.score" do
```

Comment:
I would try to organize your error scenarios separate from your "success" scenarios (so moving the first one to go along with the scenario on line 36, for ex)